docs/testfairy/app-distribution/symbols-file.md

```diff
@@ -0,0 +1,84 @@
+---
+id: symbols-file
+title: Upload/download Symbols file
+sidebar_label: Upload/download Symbols file
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Uploading Symbols Files
+
+To ensure accurate symbolication and better management of your app's debug information, it's recommended to upload mapping files (symbols) through the API. Follow these steps to attach the file during the app upload process:
+
+### 1. Prepare the Symbols File
+
+Ensure you have the mapping file (symbols) that corresponds to your app version. This file is crucial for decoding crash reports and understanding stack traces.
+
+### 2. Upload the File via API
+
+You can upload the mapping file by using the API provided by Sauce Labs. Here's a general approach:
+
+- **Endpoint:** Use the appropriate API endpoint for uploading your app. For example:
+
+- **Attachment:** Include the mapping file `symbols_file` as part of your app submission. For instance:
+
+Android:
+
+```bash
+curl https://upload.testfairy.com/api/upload \
+ -F api_key='your_api_key' \
+ -F file=@sample.apk \
+ -F symbols_file=@mappings.txt
+```
+
+IOS:
+
+```bash
+curl https://upload.testfairy.com/api/upload \
+ -F api_key='your_api_key' \
+ -F file=@sample.ipa \
+ -F symbols_file=@sample.dSYM.zip
+```
+
+### Accessing Your Symbols File
+
+After a successful upload via the API, the response will include a URL to download your Symbols file.
+
+Here’s an example of the relevant portion of the API response:
+
+```json
+{
+  "status": "ok",
+  ...
+  "symbols_download_url": "https://mobile.testfairy.com/api/xxx/projects/xxx/builds/xxx/symbols/download/",
```

Comment:
doesnt the download url contains the file extension?

Reply:
No! the download url is the api endpoint which will call the actual aws url. This url remain the same for all platforms' files